provisioning_templates/snippet/rancheros_cloudconfig.erb

```diff
@@ -9,13 +9,10 @@ hostname: <%= @host.shortname %>
 rancher:
   network:
     dns:
+<% unless @host.subnet.dns_servers.empty? -%>
```

Comment:
This is a change, but I think the previous code was incorrect. At least the `if ... or ...` statement in the previous version was redundant.

Reply:
Yes.